app/jsx/pages/BrowsePage.jsx

```diff
@@ -105,11 +106,15 @@ class AllCampuses extends React.Component {
 class AllJournals extends React.Component {
   constructor(props) {
     super(props)
-    this.state = {campusID: props.campusID,
+    this.state = {campusID: "",
                   isActive: props.isActive}
     this.changeCampus = this.changeCampus.bind(this)
   }
 
+  componentWillUnmount() {
```

Comment:
There are a couple ways to get it to retain its setting. I think the clearest is to add it as a parameter in the URL. I've not tried the more secretive ways to saving state.